stable/uchiwa/templates/NOTES.txt

```diff
@@ -9,7 +9,7 @@ Getting Started:
 
 {{- else if contains "LoadBalancer" .Values.serviceType }}
 **** NOTE: It may take a few minutes for the LoadBalancer IP to be available.                      ****
-****       You can watch the status of by running 'kubectl get svc -w {{ template "fullname" . }}' ****
+****       You can watch the status of by running 'kubectl get svc --namespace {{ .Release.Namespace }} -w {{ template "fullname" . }}' ****
```

Comment:
The stars do cause formatting issues when rendered. Would you mind removing them from the NOTES.txt in this PR?